### DISCUSSION TITLE
風景照

### DISCUSSION THREAD

**Discussion by Andrea0126** [share121/inter-knot#4845] [绝区零]
![20240914003412](https://github.com/user-attachments/assets/7a4b592f-b3e0-4f00-bddd-791bd18fb5dd)
![20240914003330](https://github.com/user-attachments/assets/2e9b23a5-87d8-4226-9309-fb368ef7c4f2)
![20240914003005](https://github.com/user-attachments/assets/62d886e0-4a07-4714-a10a-02e5621a6f7c)
做任務時看到覺得很好看就拍了，之前沒有走到這裡過，可惜不能走到下面去

**Comment by Junjun280** [comment]
好看欸！
